src/owr/OptimisticWithdrawalRecipientPectraFactory.sol

```diff
@@ -0,0 +1,119 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.19;
+
+import {OptimisticWithdrawalRecipientPectra} from "./OptimisticWithdrawalRecipientPectra.sol";
+import {LibClone} from "solady/utils/LibClone.sol";
+import {IENSReverseRegistrar} from "../interfaces/IENSReverseRegistrar.sol";
+
+/// @title OptimisticWithdrawalRecipientPectraFactory
+/// @author Obol
+/// @notice A factory contract for cheaply deploying
+/// OptimisticWithdrawalRecipientPectra.
+/// @dev This contract uses token = address(0) to refer to ETH.
```

Comment:
```suggestion

```

This is a stale comment from when the OWRs supported erc20s, and native eth was indicated by using `address(0)` for the token address, right?